feature: Active tab and badge (#3299)
filamentphp/filament: packages/forms/docs/04-layout.md

            ]),
    ])
```

> By default the first `Tab` will be marked as active, to change this behaviour you can use the `activeTab()` method of `Tabs` component to set a default active `Tab` of your own.

Reviewer: ```suggestion
The first `Tab` will be marked as active by default. You can change the default tab using the `activeTab()` method on the `Tabs` component

```
